src/bitmap.h

```diff
@@ -37,7 +37,7 @@ typedef struct _Map {
 #define MAPCLR_AT(m, n) ((m)->map[(n) >> 3] = 0)
 
 extern void map_init(Map *m, int n);
-extern void map_init_clone(Map *t, Map *s);
+extern void map_init_clone(Map *target, const Map *source);
```

Comment:
I am quite opposed to do this, it makes it inconsistent with rest of the code.

Reply:
@mlschroe What do you think? I can revert it.